src/starter/index_spec.ts

```diff
@@ -58,16 +58,6 @@ describe('starter', () => {
       );
     });
 
-    it('modifies tslint.json extends property', () => {
```

Comment:
podemos deixar este teste tmb e verificar se `tsLintBuffer` existe. antes do `expect(tsLintObject.extends).toEqual([...`

se não existir, fazer um `expect(true).toBeTruthy()` para forçar o teste a passar.